src/Map/Map.js

```diff
@@ -103,17 +103,18 @@ class Map extends React.Component {
   }
 
   getContextValue = () => {
-    const { translations } = this.props
+    const { translations, logoPosition } = this.props
 
     return {
       map: this.map,
       selectInteraction: this.selectInteraction,
-      translations
+      translations,
+      logoPosition
```

Comment:
still not sure on how this is getting added to context 🤔

Reply:
as a potential workaround for now, we could lift the basemap container & map controls by a few pixels to leave room for the logo on either side?

Reply:
I think we could just leave it as a prop for now and that way if the IA for whatever reason wants there logo on the left they can just pass it to the basemap component and it will move itself up. You cool with that?

Reply:
A prop to the components not in context